src/Event/PostMappingBuildEvent.php

```diff
@@ -30,6 +29,8 @@ public function __construct(IndexConfigInterface $indexConfig, array $mapping)
     {
         $this->indexConfig = $indexConfig;
         $this->mapping = $mapping;
+
+        parent::__construct($indexConfig->getName());
```

Comment:
I'm always confused about which name to use, should it be `$indexConfig->getElasticsearchName()`?
ping @XWB

Reply:
There is no `getElasticsearchName` method, do you mean renaming `getName()` to `getElasticsearchName()`?

Reply:
I mean this method: https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/src/Configuration/IndexConfigInterface.php#L21 vs this one https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/src/Configuration/IndexConfigInterface.php#L25

There is an abstraction of the name of the real index name that can be achieved with the bundle configuration, when using for example:
```yaml
fos_elastica:
    indexes:
        app:
            index_name: app_prod
```

`getName()` would return in this case `app` where `getElasticsearchName()` would return `app_prod`

Reply:
Good question. I have to admin I'm confused too. Perhaps we should stick with `getName()` as long as things work nicely.

Reply:
Yes, seems fair ;)